swagger.yaml

```diff
@@ -402,6 +402,98 @@ paths:
         "200":
           description: Paid status updated
 
+  "/software":
+    get:
+      summary: Get all software entries
```

Comment:
Your endpoints appear to be missing operationId and tags. you can see examples of these in the other endpoint described in this spec.

Reply:
Update all endpoints with operationId and tags